<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>918</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 20pt &quot;Ubuntu&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_dangqianshijian">
       <property name="text">
        <string>当前时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_dangqianshijianvalue">
       <property name="text">
        <string>2025/12/15 12:17:10</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="MjpgStreamViewer" name="widget_camera" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>270</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>300</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="pushButton_paizhao">
         <property name="text">
          <string>拍照</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_chakantupian">
         <property name="text">
          <string>查看图片</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_openserial">
         <property name="text">
          <string>打开串口</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_dianjizhuanxiang">
         <property name="text">
          <string>电机转向：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_dianjizhuanxiangvalue">
         <property name="text">
          <string>正转</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_dianjisudu">
         <property name="text">
          <string>电机速度：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_dianjisuduvalue">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_hongwaizhuangtai">
         <property name="text">
          <string>红外传感器状态：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_hongwaizhuangtaivalue">
         <property name="text">
          <string>遇阻</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_duojijiaodu">
         <property name="text">
          <string>舵机角度：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_duojijiaoduvalue">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_wendu">
         <property name="text">
          <string>温度：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_wenduvalue">
         <property name="text">
          <string>50°C</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_shidu">
         <property name="text">
          <string>湿度：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_shiduvalue">
         <property name="text">
          <string>30%</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_fengmingqi">
         <property name="text">
          <string>蜂鸣器状态：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_fengmingqivalue">
         <property name="text">
          <string>开</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_led">
         <property name="text">
          <string>led亮度：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_ledvalue">
         <property name="text">
          <string>20</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_controlled">
          <property name="text">
           <string>开灯</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>led亮度  ：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_led">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>舵机角度：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_servo">
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton_controlmotor">
          <property name="text">
           <string>开电机</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_motorforward">
          <property name="text">
           <string>正转</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_motorback">
          <property name="text">
           <string>反转</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>马达转速：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_motor">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>操作记录：</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView_caozuojilu">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MjpgStreamViewer</class>
   <extends>QWidget</extends>
   <header>mjpgstreamviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
